src/app/ui/dock.h

```diff
@@ -8,6 +8,7 @@
 #define APP_UI_DOCK_H_INCLUDED
 #pragma once
 
+#include "dockable.h"
```

Comment:
Please always use the full path relative to `src/`:
```suggestion
#include "app/ui/dockable.h"
```